src/acceptance/app/dynamic_policy_test.go

```diff
@@ -177,12 +204,14 @@ var _ = Describe("AutoScaler dynamic policy", func() {
 		Context("when throughput is greater than scaling out threshold", func() {
 
 			BeforeEach(func() {
-				policy = GenerateDynamicScaleOutPolicy(1, 2, "throughput", 1)
+				policy = GenerateDynamicScaleOutPolicy(1, 2, "throughput", 15)
```

Comment:
FYI: I tested this PR successfully a bit more thoroughly:
* 3 apps
* 3 service instances
* each app bound to an own service instance
* fired 50 requests per sec. against each app
* each app scaled up according to their scaling policy